### DISCUSSION TITLE
Reg: Probem with the ALAMO installation

### DISCUSSION THREAD

**Discussion by SoumyaBandyopadhyay** [solidsgroup/alamo#44] [Help] [answered]
Hello,
I tried to install ALAMO. I followed the similar steps mentioned in the 'Quick Start' . It succesfully configured, but after running 'Make'
for a while it ended up showing the following error:
![error1](https://user-images.githubusercontent.com/12543293/163107858-33d20728-fabe-485f-865a-e537a51650e4.png)
I am quite sure all the required dependencies are installed properly and the required folders, files are also present.
Still not getting the reason for this error. It will be very helpful if I get some suggestions regarding this issue.

**Comment by bsrunnels** [comment]
Try the `development` branch - it has been more recently updated. We will update the `master` branch again once AMReX 22.05 comes out. 
If that does not work, please include the output of the `./configure` script.

**Comment by bsrunnels** [comment]
A couple other things to try/check
- make sure that you have python3 installed and that `python` points to `python3` on your system (this is handy: https://launchpad.net/ubuntu/focal/+package/python-is-python3)
- after you run the configure script, go down into `./ext/amrex` and run `make` then `make install` there. If those complete successfully, you should be able to make alamo from the root.
- after switching to `development` make sure you run `make realclean`. Also, if it is there, delete the `./amrex` directory - it has been moved to `./ext/amrex`.

**Reply by SoumyaBandyopadhyay** [reply]
Thanks for the suggestions. After following the above mentioned procedure I got the following outputs:
a. running the ./configure script.
![config_res](https://user-images.githubusercontent.com/12543293/163516129-7de23f35-4553-4814-806c-8fd2d4882270.png)
  It does indicate PYTHON 3.... along will the developement branch.
b. After this I switched to /etc/amrex and successfully compilled.
![amrex_make_make_install](https://user-images.githubusercontent.com/12543293/163516469-57c096f9-09c7-4e14-abb0-cad7df690b6c.png)
c. then I switched to the developement branch and ran 'make realclean' 
![make_real_clean](https://user-images.githubusercontent.com/12543293/163516842-e2263488-987c-4925-b735-15cf71b2c125.png)
d. then I ran 'make' and again failed.
![error](https://user-images.githubusercontent.com/12543293/163517000-ba0e3b25-c436-416f-bd36-2a3a4f98a488.png)
![error2](https://user-images.githubusercontent.com/12543293/163517003-78e33d33-ea05-49ac-be32-a6b9f5aa9ad6.png)

I am not getting where am I doing the mistake? 
thanks

**Comment by bsrunnels** [comment, marked as answer]
Looks like a permissions issue. Don't use sudo for any of the commands, I think that's what is messing it up.
